hexrd/ui/calibration_slider_widget.py

```diff
@@ -290,3 +293,12 @@ def emit_update_if_polar(self):
     def reset_config(self):
         HexrdConfig().restore_instrument_config_backup()
         self.update_gui_from_config()
+
+    def update_labels(self):
```

Comment:
This should probably be called in `update_gui_from_config()` as well, so that if the user starts the program and the Euler angle convention is already like "ZXZ", then when they switch to the slider tab, it will update the labels to match.

Reply:
Fixed